src/site/stages/build/drupal/graphql/officePage.graphql.js

```diff
@@ -3,12 +3,36 @@
  * Example: /pittsburgh-health-care/events/example-event
  */
 const entityElementsFromPages = require('./entityElementsForPages.graphql');
+// Get current feature flags
+const {
+  featureFlags,
+  enabledFeatureFlags,
+} = require('./../../../../utilities/featureFlags');
 
 module.exports = `
  fragment officePage on NodeOffice {
     ${entityElementsFromPages}
     changed
     title
+    fieldDescription
+    ${
+      enabledFeatureFlags[featureFlags.FEATURE_FIELD_BODY]
```

Comment:
I'm a GraphQL n00b; please pardon the super basic question: What happens to the query if the feature flag isn't enabled?

Reply:
Your wildest dreams come true. 
But seriously, if the query hits an environment endpoint where the field has not yet been enabled / added, the build breaks. So, we are doing this check to prevent a blow up (particularly in prod). After we know that the field exists / has been deployed in Drupal all the way up to prod, we do a second release to remove the flags. We are in the process of creating a better solution - thus far, we have the Drupal logic in place, and the plan is to create the vets-website / front-end logic next sprint, @cvalarida.